assets/charts/components/inspektor-gadget/templates/psp.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: inspektor-gadget-restricted
```

Comment:
`-restricted`? From what? :joy: This is almost the most permissive policy I've seen :smile: 
```suggestion
  name: inspektor-gadget
```

Reply:
at least it can't use Secrets? :joy: